client/my-sites/sites/index.jsx

```diff
@@ -153,6 +153,24 @@ class Sites extends Component {
 			case 'site-monitoring':
 				path = translate( 'Site Monitoring' );
 				break;
+			case 'earn':
+				path = translate( 'Monetize' );
+				break;
+			case 'subscribers':
+				path = translate( 'Subscribers' );
+				break;
+			case 'themes':
+				path = translate( 'Themes' );
+				break;
+			case 'marketing':
+				path = translate( 'Marketing' );
+				break;
+			case 'import':
+				path = translate( 'Import' );
+				break;
+			case 'export':
+				path = translate( 'Export' );
+				break;
```

Comment:
Not sure how many extra you want to include but `/purchases` and `/email` are also untranslated (and nor is Jetpack Search on line 151!).

Reply:
Thanks! Added.

Jetpack Search I think is intentionally left as-is because it's a product name, and we don't translate those.

Reply:
A few more that should be translated:
customize
google-my-business
view
woocommerce-installation
store
add-ons

Reply:
Comments too!

> google-my-business

Ps. if adding this, please remember that this has renamed to Google Business Profile - see #86301